app/src/androidTest/java/com/reach5/identity/sdk/demo/MainActivityTest.kt

```diff
@@ -323,8 +347,8 @@ class MainActivityTest {
 
     @Test
     fun testSuccessfulEmailUpdate() = clientTest { client ->
-        val profile = aProfile()
-        val newEmail = anEmail()
+        val profile = ProfileSignupRequest(givenName = "Burrich", email = "burrich@gmail.com", password = "tatetutito")
+        val newEmail = "xi.mace@outlook.com"
```

Comment:
Use generated values only. I could not run the tests:
```
Reason: Bad Request Error: email_already_exists
10:41:09 V/InstrumentationResultParser: Description: Email already in use
```
Best to keep the tests homogenous too.

Reply:
You're right, I used it for my local tests but I forgot to remove it. Sorry for that ...